addons/web/static/src/views/fields/field.js

```diff
@@ -67,7 +67,7 @@ fieldRegistry.addValidation({
     },
     useSubView: { type: Boolean, optional: true },
     label: { type: [String, { value: false }], optional: true },
-    listViewWidth: { type: [Number, [Number, Number], Function], optional: true },
+    listViewWidth: { type: [Number, { type: Array, validate: (array) => array.length === 2 }, Function], optional: true },
```

Comment:
Why ?

Reply:
In debug mode i have an error that the type is not accepted

Reply:
I suppose it does not recognize the array with the [Number, Number] ?